### ISSUE TITLE
obsidian integration context menu not appearing on chrome

### ISSUE THREAD

**Issue by tlonic** [deathau/markdownload#166]
Using Chrome 106.0.5249.91 (Official Build) (64-bit) on Archlinux with the web store extension 3.2.1. 

My configuration is as follows:

> {
  "backmatter": "",
  "bulletListMarker": "-",
  "codeBlockStyle": "fenced",
  "contextMenus": true,
  "disallowedChars": "[]#^",
  "downloadImages": false,
  "downloadMode": "contentLink",
  "emDelimiter": "_",
  "fence": "```",
  "frontmatter": "---\npage-title: \"{pageTitle}\"\ndate: \"{date:YYYY-MM-DDTHH:mm:ss}\" \nurl: {baseURI}\n---\n",
  "headingStyle": "atx",
  "hr": "___",
  "imagePrefix": "{pageTitle}/",
  "imageRefStyle": "inlined",
  "imageStyle": "originalSource",
  "includeTemplate": true,
  "linkReferenceStyle": "full",
  "linkStyle": "inlined",
  "mdClipsFolder": null,
  "obsidianFolder": "misc-reading",
  "obsidianIntegration": true,
  "obsidianVault": "obsidian",
  "saveAs": false,
  "strongDelimiter": "**",
  "title": "{pageTitle}",
  "turndownEscape": true
}

The option to import to Obsidian does not appear: 
![image](https://user-images.githubusercontent.com/2394967/193971596-7990ab12-00d2-490f-9962-b92522d0e670.png)

**Comment by fahran** [comment]
Possibly I know the answer to this one, as I had to go hunting yesterday.

The "send to obsidian" option doesn't appear when you right click the extension, but rather when you right-click the page itself.

https://user-images.githubusercontent.com/2371903/196658591-01e232d2-c260-47e7-9bdd-b4921d3b4087.mp4

**Comment by aancw** [comment]
@tlonic as @fahran said, for now the obsidian integration is only available via context menu. As I'm not expert on js and web ext, I will try my best to learn how to add the obsidian feature on obsidian icon click.

**Comment by Gr-Nobel** [comment]
@aancw i click the option :" send to opsidian" but just obsidian opens no markdownfile.

**Comment by Captain-Leftovers** [comment]
Yeah same here it just copies the selection or tab it doesn't create a file or download anything

**Comment by Gr-Nobel** [comment]
@Captain-Leftovers Install the advance uri plugin.
Then go to plugin option . Check the obsidian integration option . Then add vault name and file you want to add into. For me its Resources/Web.

Keep obsidian open then click open with obsidian it should work.

**Comment by Captain-Leftovers** [comment]
> @Captain-Leftovers Install the advance uri plugin. Then go to plugin option . Check the obsidian integration option . Then add vault name and file you want to add into. For me its Resources/Web.
> 
> Keep obsidian open then click open with obsidian it should work.

So the problem i missed in my case was that i use Obsidian on my windows and my linux machine and when i do that the extension is  performing sync on the other machine  The problem is the path to the vault is different in Linux and Windows so the sync is making it not work on two machines
i know i can disable extensions sync but the problem with that is that i only cna disable all extensions sync not just MarkDownload
pls correct me if im wrong
thank you

**Comment by Flashwalker** [comment]
When installing an extension as an unzipped source, the context menu items of Onbsidian disappear

**Comment by Flashwalker** [comment]
> So the problem i missed in my case was that i use Obsidian on my windows and my linux machine and when i do that the extension is performing sync on the other machine The problem is the path to the vault is different in Linux and Windows so the sync is making it not work on two machines i know i can disable extensions sync but the problem with that is that i only cna disable all extensions sync not just MarkDownload pls correct me if im wrong thank you

Are you referring to the case of downloading an md file to a specified folder?
Since saving via Obsidian URI does not require an absolute path, only a vault name and a folder name.
